docs/guides/openshift/local.md

```diff
@@ -19,7 +19,7 @@ Red Hat OpenShift Local at present is only supported on AMD64 and Intel 64 proce
 And the following operating system requirements:
 
 - Microsoft Windows 10 Fall Creators Update (version 1709) or later
-- macOS 11 Big Sur or later
+- MacOS 11 Big Sur or later
```

Comment:
Apple uses macOS

The whole system requirements section should be removed or just a link provided to the official system requirements.

Reply:
Done